colors/gruvbox-material.vim

```diff
@@ -72,8 +72,13 @@ else
     endif
   endif
 endif
-call gruvbox_material#highlight('IncSearch', s:palette.bg0, s:palette.bg_red)
-call gruvbox_material#highlight('Search', s:palette.bg0, s:palette.bg_green)
+if has('nvim')
+  call gruvbox_material#highlight('IncSearch', s:palette.bg0, s:palette.bg_red)
```

Comment:
I'm not sure whether this is needed, but it looks pretty similar to `Search` so I thought to include it too. Does it make sense?